x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/rule_actions_overflow/index.tsx

```diff
@@ -177,6 +181,33 @@ const RuleActionsOverflowComponent = ({
             >
               {i18nActions.MANUAL_RULE_RUN}
             </EuiContextMenuItem>,
+            ...(rule.rule_source.type === 'external' && rule.rule_source.is_customized === true // Don't display action if rule isn't a customized prebuilt rule
```

Comment:
The pattern `rule_source.type === 'external' &&rule_source.is_customized === true or false` is used multiple time throughout the codebase. It looks like we should introduce helper functions like `isCustomizedPrebuiltRule` and `isNonCustomizedPrebuiltRule`.